tools/NewModuleScript/MapViewModelTemplate.kt

```diff
@@ -28,6 +28,7 @@ import com.esri.arcgismaps.sample.sampleslib.components.MessageDialogViewModel
 import kotlinx.coroutines.launch
 
 class MapViewModel(application: Application) : AndroidViewModel(application) {
+    // TODO - The ArcGISMap only needs to be wrapped in a MutableState object if it will change in this sample
```

Comment:
Could be
"TODO - delete mutable state when the map does not change or the screen does not need to observe changes".